### PR TITLE
Geofence & Rally Point definitions

### PR DESCRIPTION
**Pull request by bkueng**
Continuation of #681. Additional changes: cc5b567

I think this is good go merge. If there are more things coming up during implementation, we can iterate on that.
@DonLakeFlyer @julianoes @LorenzMeier

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1445,6 +1445,50 @@
...
+      <entry value="5000" name="MAV_CMD_NAV_FENCE_RETURN_POINT">
+        <description>Fence return point. There can only be one fence return point.
+        </description>
+        <param index="1">Reserved</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4">Reserved</param>
+        <param index="5">Latitude</param>
+        <param index="6">Longitude</param>
+        <param index="7">Altitude</param>
```
Comment (julianoes) [inline message_definitions/v1.0/common.xml]: Is this relative or absolute altitude?

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -1445,6 +1445,50 @@
...
+        <param index="1">Polygon vertex count</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4">Reserved</param>
+        <param index="5">Latitude</param>
+        <param index="6">Longitude</param>
+        <param index="7">Reserved</param>
+      </entry>
+      <entry value="5100" name="MAV_CMD_NAV_RALLY_POINT">
+        <description>Rally point. You can have multiple rally points defined.
```
Comment (julianoes) [inline message_definitions/v1.0/common.xml]: What rally are we doing? Is there an explanation what a rally is yet?

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -1445,6 +1445,50 @@
...
+        </description>
+        <param index="1">Reserved</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4">Reserved</param>
+        <param index="5">Latitude</param>
+        <param index="6">Longitude</param>
+        <param index="7">Altitude</param>
```
Comment (julianoes) [inline message_definitions/v1.0/common.xml]: Again, relative or absolute?

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (bkueng) [inline message_definitions/v1.0/common.xml]: That's defined by `MAV_FRAME`, used in mavlink_mission_item, in which this command is embedded. From what I see all mission items are handled this way, so IMO does not need further clarification here.

File: message_definitions/v1.0/common.xml (hunk 2 above)
Comment (bkueng) [inline message_definitions/v1.0/common.xml]: is https://github.com/mavlink/mavlink/pull/683/commits/cc5b567d1d2fff827331dd0b9c5e3a7d147a46ac#diff-ceacc1e2390186650146a45be8fcb691R2125 enough?

Comment (julianoes) [inline message_definitions/v1.0/common.xml]: Got it, sure.

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (julianoes) [inline message_definitions/v1.0/common.xml]: Aaaha, awesome, thanks.

**Comment by DonLakeFlyer** [comment]
I guess I have rights to merge this in. I've never merged anything in this repo. are the power that be ok with me doing that so I can get going on this?

**Pull request merged**